tools/wptrunner/wptrunner/wptcommandline.py

```diff
@@ -323,9 +323,14 @@ def create_parser(product_choices=None):
                              help="Command-line argument to forward to the "
                                   "Sauce Connect binary (repeatable)")
 
-    webkit_group = parser.add_argument_group("WebKit-specific")
+    webkit_group = parser.add_argument_group("Epiphany and WebKit-specific")
```

Comment:
Do the flatpak things actually apply to anything other than Epiphany? Seems like there could be a seperate group just for these if not.